lib/news_scraper/trainer/presets_selector.rb

```diff
@@ -37,20 +37,23 @@ def preset_results(payload, presets, data_type)
         return {} unless presets
 
         scrape_details = blank_scrape_details
+        scrape_patterns = YAML.load_file(NewsScraper::Constants::SCRAPE_PATTERN_CONFIG_FILE)
```

Comment:
Why is this being reloaded again? 

It's already reloaded in the initializer, which should be sufficient since we only require 1 reload per uri